paddle/inference/example.cc

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include <time.h>
+#include <iostream>
+#include "paddle/inference/inference.h"
+
+int main(int argc, char* argv[]) {
+  std::string dirname =
+      "/home/work/liuyiqun/PaddlePaddle/Paddle/paddle/inference/"
```

Comment:
Do not use absolute/personal path

Reply:
Maybe you can upload this model, and download it by verifying MD5.

Reply:
> Do not use absolute/personal path

Done.

> Maybe you can upload this model, and download it by verifying MD5.

I think the current version of model is not stable and is not suitable to upload for testing.